inductiva/molecules/scenarios/protein_solvation/protein_solvation.py

```diff
@@ -43,52 +43,26 @@ def __init__(
     def simulate(
             self,
             simulator: Simulator = GROMACS(),
-            working_dir: Optional[Path] = None,
+            output_dir: Optional[Path] = None,
             simulation_time: float = 10,  # ns
-            integrator: str = "md",
+            integrator: Literal["md", "sd", "bd"] = "md",
```

Comment:
You're right and even the pipeline that I am creating is very specific to GROMACS. I wanted to follow the structure of the other scenarios, that's why I did it like this, but should I simplify and do it just for GROMACS?

Reply:
A future exploration to keep a few things in mind: 
- Does this integrator is specific to GROMACs or other simulators use it under a different name? In case it is not specific, you can follow a structure similar to the link above by providing more descriptive names to the integrators and then connecting them to each specific simulator.
- Is the choice of integrator specific for the scenario we are using here. That is, will the user really need to select a different one or can we keep it fixed for now?

I think this are important to explore and will keep the scenario more agnostic without to much changes. However, one can keep it "specific" for GROMACs for now and when adding a new simulator to molecular dynamics and this scenario change it accordingly.

Reply:
I think that other simulators will have a similar parameter to specify the integrator, however I haven't explored enough to say for sure. The choice of the integrator is not specific to this scenario, I just left it because I wanted to explore what is the best one.